ginkgo_me/BBQ_gotest_test.go

```diff
@@ -0,0 +1,152 @@
+package ginkgo_me
+
+import (
+	"testing"
+
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/assert"
+
+	mock_beeffarm "github.com/artiomgiza/test_me/pkgs/beef-farm/mock"
+	mock_chickenfarm "github.com/artiomgiza/test_me/pkgs/chicken-farm/mock"
+	mock_coalfarm "github.com/artiomgiza/test_me/pkgs/coal-farm/mock"
+	mock_mangalstore "github.com/artiomgiza/test_me/pkgs/mangal-store/mock"
+	"github.com/golang/mock/gomock"
+)
+
+var (
+	beefFarmMock    *mock_beeffarm.MockProvider
+	chickenFarmMock *mock_chickenfarm.MockProvider
+	mangalStoreMock *mock_mangalstore.MockProvider
+	coalFarmMock    *mock_coalfarm.MockProvider
+
+	subject Provider
+)
+
+func TestCoolPriceCalculator2(t *testing.T) {
+	t.Run("#CalculatePrice", func(t *testing.T) {
+		var (
+			numberOfPeople int
+			totalPrice     int
```

Comment:
why do we need this definition here?
you might use `:=` in line 52 (and others) and remove this from here. 
wdyt?

Reply:
same for others